docs/pages/application-access/okta/introduction.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Okta integration with Application Access
+description: How to set up Okta integration with Teleport Application Access.
+---
+
+Teleport allows applications housed within Okta to be integrated with the
+Application Access service. This will result in Okta applications showing up
+in Teleport's applications list in the UI and CLI, and it will further allow
+for the user of Teleport's role based access control to control access within
+Okta.
+
+Users will be able request and be granted access to applications and groups
+within Okta by requesting access to them within Teleport.
+
+## How does the Okta integration work?
+
+Teleport provides an Okta service that is responsible for dealing with all
```

Comment:
If the Okta service is a first-class Teleport service in the way the Application Service, SSH Service, Discovery Service, etc. are, I think we should capitalize the name of the service to be consistent with other Teleport services.